src/applications.md

```diff
@@ -24,5 +25,7 @@ Applications of Rust:
 * [_nalgebra_](https://nalgebra.org/docs/) - a linear algebra library
 * [Prusti user guide](https://viperproject.github.io/prusti-dev/user-guide/) - a Rust verifier
 * [Serde](https://serde.rs/) - **ser**ialize and **de**serialize Rust data structures
+* [_Solana_](https://docs.solana.com/) - web-scale blockchain for decentralized apps
+* [_tantivy_](https://tantivy-search.github.io/examples/basic_search.html) - a full text search engine library
 * [Tealdeer User Manual](https://dbrgn.github.io/tealdeer/) - implementation of [tldr](https://github.com/tldr-pages/tldr) (help like `man`, but more approachable).
```

Comment:
@kw-fn I don't think the above two should be included because they don't even look slightly like they are in mdbook format. My rationale for restricting to "books" is that this is the Little Book of Rust _Books_. If we drop that restriction, we'll end up listing hundreds of Rust web sites and this book will lose its attraction.

Reply:
I removed these two.

Reply:
Actually I think these are both special cases from a couple points-of-view, I can provide some context for you:

1. They can be seen reference implementations of the services they provide, and,
2. they use novel and sophisticated approaches to documentation.

It's probably more arguable that adding `mdbook` formatted content would lead to bloat since that is the default being used nearly everywhere. Here the consideration was the novelty of the custom documentation and the relative importance of each crate within its ecosystem.

Reply:
> Actually I think these are both special cases from a couple points-of-view, I can provide some context for you:
> 
>     1. They can be seen reference implementations of the services they provide, and,
> 
>     2. they use novel and sophisticated approaches to documentation.
> 
> 

I had another look at these two. I agree that Solana is close to book format and I'd be happy to have it included. But the tantivy site isn't anything like a book really and I feel it would dilute the LBORB concept of being a collection of _books_.

> It's probably more arguable that adding `mdbook` formatted content would lead to bloat since that is the default being used nearly everywhere. Here the consideration was the novelty of the custom documentation and the relative importance of each crate within its ecosystem.

The concept of LBORB, which got hundreds of positive upvotes on [Reddit](https://www.reddit.com/r/rust/comments/kwiwb8/the_little_book_of_rust_books/), was to collect together mdbook format books. I'd like to maintain that concept and I hope we can align on that.

Reply:
In some sense I can see there is value in denoting which links don't follow along with a certain predominant template. On the other hand I do not see any utility in excluding content solely based on it not conforming exactly enough to some novel criteria. 

As an aside here I do think that tantivy is formatted in a way that does meet the criteria of being a book, which I think is to say that it encodes logical separation based on chapters, sections, and sub-sections.

Thanks for your efforts in maintaining the standards.

Reply:
> In some sense I can see there is value in denoting which links don't follow along with a certain predominant template. On the other hand I do not see any utility in excluding content solely based on it not conforming exactly enough to some novel criteria.

I think we'll have to agree to differ on this point.

> 
> As an aside here I do think that tantivy is formatted in a way that does meet the criteria of being a book, which I think is to say that it encodes logical separation based on chapters, sections, and sub-sections.

That is true, but I would say the books in LBORB generally try quite hard to be self-contained and define their terms, another facet of a book in addition to its structure into chapters etc.. The tantivy document is really just an example which assumes quite a bit of prior understanding of tantivy (such as what the name even means), so I don't think it falls into the "hono[u]rable mention" category of non-mdbook books which deserve including because of their exceptional value.

> 
> Thanks for your efforts in maintaining the standards.

You're welcome. It has been helpful for you to challenge the standards a little as it helps to clarify the criteria for inclusion.